main.py

```diff
@@ -1,11 +1,38 @@
 import init_django_orm  # noqa: F401
 
-from db.models import Race, Skill, Player, Guild
+import json
+from datetime import datetime
+from db.models import Race, Skill, Guild, Player
 
+def main():
+    # Read data from players.json
+    with open('players.json', 'r') as file:
+        players_data = json.load(file)
 
-def main() -> None:
-    pass
+    for player_data in players_data:
+        # Check if the race already exists, otherwise create it
+        race, created_race = Race.objects.get_or_create(name=player_data['race_name'], defaults={'description': player_data.get('race_description', '')})
 
+        # Check if the guild already exists, otherwise create it
+        guild, created_guild = Guild.objects.get_or_create(name=player_data['guild_name'], defaults={'description': player_data.get('guild_description', '')})
 
-if __name__ == "__main__":
+        # Create skills for the race (if they don't exist)
+        for skill_data in player_data['skills']:
+            Skill.objects.get_or_create(
+                name=skill_data['name'],
+                bonus=skill_data['bonus'],
+                race=race
+            )
+
+        # Create player
+        Player.objects.get_or_create(
+            nickname=player_data['nickname'],
+            email=player_data['email'],
+            bio=player_data['bio'],
+            race=race,
+            guild=guild,
+            created_at=datetime.now()  # Set the creation date to now
```

Comment:
Using `datetime.now()` for the `created_at` field might not be necessary since `auto_now_add=True` is already set in the model definition. Consider removing this field from `get_or_create` to rely on the model's default behavior.